Docs/AcpiSamples/SSDT-EC-USBX.dsl

```diff
@@ -51,7 +52,20 @@ DefinitionBlock ("", "SSDT", 2, "ACDT", "SsdtEC", 0x00001000)
             }
         }
     }
-    **/
+    
+    if the STA method exists, use
+    
+    External (_SB_.PCI0.LPCB.EC0_._STA, UnknownObj)
```

Comment:
Better make this a separate comment somewhat like:
```C
/*
 * Uncomment the following to disable EC if the STA method exists.
 */

/**

…

 **/
```

Reply:
ok, I'm working on ssdt plug, let's make a new request? or do I update you here?

Reply:
the change you propose is ok, it reads better, easier to understand